cmd/booster-bitswap/init.go

```diff
@@ -115,13 +114,3 @@ var initCmd = &cli.Command{
 		return err
 	},
 }
-
-var versionCmd = &cli.Command{
```

Comment:
No need for this, `--version` flag is supported out of the box with `urfave/cli`